### PR TITLE
Create user model

### PR DESCRIPTION
**Pull request by Blokkinen**
PT Story: https://www.pivotaltracker.com/story/show/133185993

Changes proposed in this pull request:

- Install Devise
- Generate a User model
- Add link to "sign up" on index page
- Fix css for the "sign up", "log in" and "forgot password" page
- Tests ofcourse!
- Add "erb2haml" gem

What I have learned working on this feature: 

- Thats Devise was really easy to set up
- That you can't name a model the same as the app
- That making it pretty wasn't very time consuming or hard either (With some pointers from Suss)

Screenshots: 
skarmavbild 2016-10-26 kl 22 15 37

Ready for review!

### CODE REVIEW

**Comment by tochman** [comment]
Great job Viktoria. More Haml please. 😉

**Comment by thesuss** [comment]
Look into https://github.com/dhl/erb2haml

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  devise_for :users
```
Comment (AmberWilkie) [inline config/routes.rb]: I'd like to see this inside the `quizmaster` namespace

File: features/quizmaster/register_quizmaster.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  Then I should see "Play the game"
+  When I click the "Log in as Quizmaster" link
+  Then I should be on the Log in page
+  And I click the "Sign up" link
+  Then I should see "Sign up"
+  When I fill in "Email" with "viktoria@quizmaster.com"
+  And I fill in "Password" with "password"
+  And I fill in "Password confirmation" with "password"
+  And I click the "Sign up" button
+  Then I should see "Welcome! You have signed up successfully."
```
Comment (AmberWilkie) [inline features/quizmaster/register_quizmaster.feature]: I'd like to see sad path for user registration and login. What happens when I put in the wrong password or try to sign up with no email?

**Pull request merged**